app/src/main/java/com/android/unio/ui/user/UserProfile.kt

```diff
@@ -243,6 +255,14 @@ fun UserProfileScreenContent(navigationAction: NavigationAction, user: User) {
                 AssociationSmall(it) { navigationAction.navigateTo(Screen.ASSOCIATION_PROFILE) }
               }
             }
+          } else {
+            Text("You are not member of any association yet", style = AppTypography.bodySmall)
```

Comment:
If you are not member of any association, you will be pushed to claim one or join one. 
To be very useful, we should implement the "join" management which is outside the scope of this PR. However, some people also thought we should not push people to join or claim an association as the majority of users won't be in an association in all probability. We will discuss it in next meeting, and I will refactor it accordingly.